test/preflight/apps_v2_integration_test.go

```diff
@@ -1,494 +0,0 @@
-//go:build integration
-// +build integration
-
-package preflight
-
-import (
-	"errors"
-	"fmt"
-	"net/http"
-	"os"
-	"path/filepath"
-	"reflect"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/stretchr/testify/require"
-	"github.com/superfly/flyctl/api"
-	"github.com/superfly/flyctl/internal/appconfig"
-	"github.com/superfly/flyctl/test/preflight/testlib"
-)
-
-func TestAppsV2Example(t *testing.T) {
```

Comment:
I don't think this is a nomad related test, this just tests some apps v2 functionality. We could maybe rename it to TestAppExample though

Reply:
i think that goes for every test in this file

Reply:
Oh they were removed by e80f21b6cffca9bc2db1e1b65409c146f0d1defc already